docs/content/latest/yugabyte-cloud/cloud-quickstart/qs-connect.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Connect to your cluster
+linkTitle: Connect to your cluster
+description: Connect to your cluster using the cloud shell.
+headcontent:
+image: /images/section_icons/index/quick_start.png
+menu:
+  latest:
+    identifier: qs-connect
+    parent: cloud-quickstart
+    weight: 200
+isTocNested: true
+showAsideToc: true
+---
+
+After [creating a free cluster](../qs-add/), the easiest way to connect to it is to use the cloud shell.
+
+The cloud shell allows you to connect to and interact with your Yugabyte Cloud cluster from your browser. You have the option of using the following CLIs in the cloud shell:
+
+- [ysqlsh](../../../admin/ysqlsh/) - YSQL shell for interacting with YugabyteDB using the [YSQL API](../../../api/ysql) 
+- [ycqlsh](../../../admin/ycqlsh/) - YCQL shell, which uses the [YCQL API](../../../api/ycql)
+
+## Connect to your cluster using the cloud shell
+
+To use `ysqlsh` to create and manage YugabyteDB databases and tables in your Yugabyte Cloud cluster, do the following:
+
+1. On the **Clusters** page, click **Connect** to display the **Connect to Cluster** dialog.
```

Comment:
Here , on the `clusters` page , I first need to `click the cluster` itself after which I can click `connect` and then the `Connect to Cluster` dialog appears. So you probably need to include this detail.